src/sentry/runner/commands/killswitches.py

```diff
@@ -110,6 +110,8 @@ def _push(killswitch_name, infile, yes):
             "Should the changes be applied?", default=False, show_default=True, abort=True
         )
     options.set(killswitch_name, new_option_value)
+    if callback := killswitches.ALL_KILLSWITCH_OPTIONS[killswitch_name].on_change:
+        callback(option_value, new_option_value)
```

Comment:
Maybe this could also be implemented as a signal handler on the `Option` model, but I personally prefer an independent trigger mechanism for kill switches.